identity/app/views/profile/privacyForm.scala.html

```diff
@@ -104,15 +104,21 @@ <h2>Thank you</h2>
 @profile.emailPrefs(IdentityPage("/email-prefs", "Email preferences"), emailPrefsForm, emailSubscriptions, availableLists, idRequest, idUrlBuilder)
 
 @* CHANNELS *@
-@if(IdentityShowCommunicationChannelConsents.isSwitchedOn && communicationChannelsForUser(user).nonEmpty) {
+@if(IdentityShowCommunicationChannelConsents.isSwitchedOn) {
     <hr class="manage-account-divider" />
 
     <fieldset class="fieldset fieldset--manage-account-noborder">
         <div class="fieldset__heading">
-            <h2 class="form__heading">How can we get in touch with you?</h2>
+            <h2 class="form__heading">How else can we get in touch with you?</h2>
         </div>
         <div class="fieldset__fields">
-            @channelConsentForm
+            @if(communicationChannelsForUser(user).nonEmpty) {
+                @channelConsentForm
+            } else {
+                <p class="identity-title-explainer">
+                    Complete your <a href="@idUrlBuilder.buildUrl("/account/edit", idRequest)" data-link-name="identity : email : add-channel-info">Account details</a> with your phone number or your home address to your to be able to opt in to receiving phone, SMS, and post communications from the Guardian.
```

Comment:
How about:

_Update your account details with your phone number or home address to be able to opt in to receiving phone, SMS, and post communications from the Guardian._